cube.py

```diff
@@ -0,0 +1,37 @@
+"""
+This module defines a Cube class, and prints a list of 25
+randomly chosen faces of the cube.
+"""
+
+import random
+
+
+# Number of moves in scramble algorithm.
+K = 25
+
+
+""" A list of possible modifications on a move. To be appended to a
+face value like so: 'U2'.
+"""
+MODS = ["2", "'"]
+
+
+class Cube:
+    """
+    Class attribute: A list of faces on the cube,
+    U : Up, D : Down (bottom face), and so on.
+    """
+    FACES = ['U', 'D', 'F', 'R', 'B', 'L']
+
+    def __init__(self, order):
+        self.order = order
+
+    # Instance method randomly chooses from FACES, K times
+    def __str__(self):
+        scramble_alg = random.choices(Cube.FACES, k = K)
```

Comment:
This probably shouldn't be done in the `__str__` method. Maybe you could have another method that generates a scramble and returns it. You could always print that. Remember that this class is mostly for generating data. Any representations of that data are up to the front end.

If this class was saving a scramble state, then maybe it would make sense to have `__str__` return it, but it is very odd that you get a different unexpected answer every time you print this object.

Reply:
It started out as a scramble method but I was having trouble printing it until I changed it to `__str__`. But I guess if we are using flask I shouldn't print anything right?

Why is it odd to get a different unexpected answer every time? It should be a different scramble every time right?

Reply:
Yeah, you can print when you are debugging, but you shouldn't be trying to merge debug code. Just print whatever method returns a scramble.

You aren't using `__str__` for its intended purpose. I would be confused if I tried to print my Cube and got a scramble instead. Not only that a different scramble every time. Picture if you tried to print a musical instrument you just made and instead of it telling you its tuning/number_of_strings/number_of_frets/etc it was all "LOL E Mixolydian" and you were all "wait what?" and it was "Fuck you A flat Major."